pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -439,15 +439,34 @@ func (s *DiscoveryServer) configz(w http.ResponseWriter, req *http.Request) {
 	fmt.Fprint(w, "\n{}]")
 }
 
+const (
+	authnHTTP       = 1
```

Comment:
Suggest to add a `typdef` as opposed to int directly.

And if you `typedef authProtocol string`, you don't need the conversion function below.